src/app/financial-chart/services/financial-data.service.ts

```diff
@@ -0,0 +1,21 @@
+import { Injectable } from "@angular/core";
+
+@Injectable()
+export class FinancialDataService {
+
+    public getData(): any[] {
```

Comment:
I think you may want to have more data here.

Reply:
I've got 5 years worth of data ready to swap in.  How much are you thinking?

Reply:
Hmm... not sure. why don't I just approve it in this state, you maybe we need to start a conversation as to how much to throw in there with stakeholders.